examples/website/geojson/app.js

```diff
@@ -30,14 +31,19 @@ export const COLOR_SCALE = scaleThreshold()
     [128, 0, 38]
   ]);
 
-const LIGHT_SETTINGS = {
-  lightsPosition: [-125, 50.5, 5000, -122.8, 48.5, 8000],
-  ambientRatio: 0.2,
-  diffuseRatio: 0.5,
-  specularRatio: 0.3,
-  lightsStrength: [2.0, 0.0, 1.0, 0.0],
-  numberOfLights: 2
-};
+const ambientLight = new AmbientLight({color: [255, 255, 255], intensity: 1.0});
+const pointLight = new PointLight({
```

Comment:
Let's test the lighting effects in the layer browser, not the examples. The examples should be minimal and self-explanatory.

After removing these custom lights, this example can be a good verification of whether the default light settings are reasonable.